src/mod/channels.mod/channels.h

```diff
@@ -38,7 +38,8 @@
 #define CHAN_RESETBANS    0x08
 #define CHAN_RESETEXEMPTS 0x10
 #define CHAN_RESETINVITED 0x20
-#define CHAN_RESETALL     0x3F
+#define CHAN_RESETAWAY    0x40
```

Comment:
please call it RESETWHO or similar, this can be used for many more things (like accounts)

Reply:
RESETWHO is already in use 5 lines above this. It updates the entire member record from a 352 message, which involves clearing out the entire channel list and resetting it- to include IDLE times maintained by eggdrop. The RESETAWAY flag does not clear the record like WHO does, only update the away field of the user record (thereby preserving the idle time). The resetchan command is structured to be used for many more things via the use of flags, you can see those already currently available in the tcl-command doc. Something like account-notify could/should be easily fit into that structure